src/main/java/pl/nextleveldev/smart_route/route/RouteService.java

```diff
@@ -0,0 +1,12 @@
+package pl.nextleveldev.smart_route.route;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class RouteService {
+    public List<Object> searchRoute(final String startLocation, final String destination) {
```

Comment:
I would return some dummy dto, but not object
thats why: https://github.com/allegro/restapi-guideline?tab=readme-ov-file#wrap-collection-in-object